yarn-project/end-to-end/src/e2e_deploy_contract.test.ts

```diff
@@ -92,6 +92,30 @@ describe('e2e_deploy_contract', () => {
     }
   }, 30_000);
 
+  /**
+   * Verify that we can produce a rollup with multiple txs
+   */
+  // TODO: Unskip after nullifier tree errors have been dealt with
+  it.skip('should deploy multiple contracts in a single block', async () => {
+    const deployer = new ContractDeployer(abi, aztecRpcServer);
+    await (node as unknown as TestAztecNode).sequencer.stop();
```

Comment:
I don't think this sort of test belongs here and it also means we have exposed the internals of the node and having to restart the sequencer etc. These aren't things that an Aztec 3 developer should be concerned with or even have access to.

Tests that verify the sequencer correctly processes blocks of txs should be at the sequencer level where we have the required control.

Reply:
In the meantime, I'm deleting the test (it was skipped after all). @PhilWindle do you think we can merge this?